core/src/test/java/org/jboss/set/mjolnir/archive/mail/report/InvalidResponsiblePersonTableTestCase.java

```diff
@@ -0,0 +1,96 @@
+package org.jboss.set.mjolnir.archive.mail.report;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.apache.deltaspike.core.util.ArraysUtils;
+import org.apache.deltaspike.testcontrol.api.junit.CdiTestRunner;
+import org.jboss.set.mjolnir.archive.domain.RegisteredUser;
+import org.jboss.set.mjolnir.archive.ldap.LdapDiscoveryBean;
+import org.jboss.set.mjolnir.archive.ldap.LdapScanningBean;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.inject.Inject;
+import javax.naming.NamingException;
+import javax.persistence.EntityManager;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.doReturn;
+
+@RunWith(CdiTestRunner.class)
+public class InvalidResponsiblePersonTableTestCase {
+
+    @Rule
+    public WireMockRule wireMockRule = new WireMockRule(8089);
+
+    @Inject
+    private EntityManager em;
+
+    @Inject
+    private LdapScanningBean ldapScanningBean;
+
+    @Inject
+    private InvalidResponsiblePersonTable invalidResponsiblePersonTable;
+
+    @Inject
+    private LdapDiscoveryBean ldapDiscoveryBeanMock;
+
+    @Before
+    public void setup() throws NamingException {
+
+        HashMap<Object, Object> ldapUsersMap = new HashMap<>();
+        ldapUsersMap.put("Responsible guy", false);
+        ldapUsersMap.put("Responsible guy1", false);
+        ldapUsersMap.put("non-existing", true);
```

Comment:
IIRC we pretend this user does actually exists, so I would change the "non-existing" name to something else, to avoid confusion.